src/Database/DBStructure.php

```diff
@@ -48,6 +48,22 @@ class DBStructure
 	 */
 	private static $definition = [];
 
+	/**
+	 * Set a database version to trigger update functions
+	 *
+	 * @param string $version
+	 * @return void
+	 */
+	public static function setDatabaseVersion(string $version)
+	{
+		if (!is_numeric($version)) {
+			throw new \Asika\SimpleConsole\CommandArgsException('The version number must be numeric');
+		}
+
+		DI::config()->set('system', 'build', $version);
+		echo DI::l10n()->t('The database version had been set to %s.', $version);
```

Comment:
Please don't `echo` anything anywhere, we have several ways of displaying the information depending on the context. If you're in the command-line context, then the Console command class should create that output, not the underlying class. Please return a boolean with the success of the operation instead.

Reply:
Is there an example somewhere? Because I just did it like it had been done in other parts of the code.

Reply:
An example of not echoing strings inside non-module classes? It loos like `Database\DBStructure` and `Database\View` are the only two exceptions to this unwritten rule. So I understand you reproduced what was in the same file, but ideally it shouldn't be done at all in these two classes as well.

In this case, it's pretty easy since there aren't several outcomes that would require more complex Exception throwing and catching.